addons/web/views/report_templates.xml

```diff
@@ -406,14 +406,14 @@
         </div>
 
         <div t-attf-class="footer o_company_#{company.id}_layout {{report_type != 'pdf' and 'mt-auto'}}">
-            <div class="o_footer_content row border-top pt-2">
-                <div class="col-8">
+            <div class="o_footer_content border-top pt-2" t-attf-class="{{'d-flex' if is_html_empty(company.report_header) else 'row' }}">
```

Comment:
note that if t-att[f]-class is set on a node, the class attribute will not be rendered
(this comment is generic for all attributes)

`ir_qweb.py:def _compile_directive_att`